plugins/flatpak/gs-flatpak.c

```diff
@@ -3569,6 +3576,7 @@ gs_flatpak_finalize (GObject *object)
 
 	g_free (self->id);
 	g_object_unref (self->installation);
+	g_ptr_array_unref (self->installed_refs);
```

Comment:
It might be possible for `self->installed_refs == NULL` at this point (if there are no apps to refine, for example), so this should be `g_clear_pointer (&self->installed_refs, g_ptr_array_unref)`

Reply:
Good catch, updated.